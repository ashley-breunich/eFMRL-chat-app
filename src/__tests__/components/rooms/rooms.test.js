import React from 'react';
import renderer from 'react-test-renderer';
import Rooms from 'components/rooms/rooms.js';

describe('<Room />', () => {
  it('Room is alive at application start - h2', () => {
    let app = shallow(<Rooms />);
    expect(app.find('h2').exists()).toBeTruthy();
  });

  it('Room is alive at application start - div', () => {
    let app = shallow(<Rooms />);
    expect(app.find('div').exists()).toBeTruthy();
  });

  it('Room is alive at application start - button', () => {
    let app = shallow(<Rooms />);
    expect(app.find('button').exists()).toBeTruthy();
  });

  it('Header renders correctly', () => {
    const header = renderer.create(<Rooms />).toJSON();
    expect(header).toMatchSnapshot();
  });
});
