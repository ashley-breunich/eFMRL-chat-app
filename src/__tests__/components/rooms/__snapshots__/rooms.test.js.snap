// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Room /> Header renders correctly 1`] = `
Array [
  <h2
    className="roomTitle"
  >
    Rooms
  </h2>,
  <div
    className="buttonGroup"
  >
    <button
      style={
        Object {
          "fontWeight": "normal",
        }
      }
      value="general"
    >
      general
    </button>
    <button
      style={
        Object {
          "fontWeight": "normal",
        }
      }
      value="sports"
    >
      sports
    </button>
    <button
      style={
        Object {
          "fontWeight": "normal",
        }
      }
      value="coding"
    >
      coding
    </button>
    <button
      style={
        Object {
          "fontWeight": "normal",
        }
      }
      value="fashion"
    >
      fashion
    </button>
  </div>,
]
`;
